### ISSUE TITLE
[P2][Sourcing][Accessibility] Some elements are not focusable using TAB inside table

### ISSUE THREAD

**Issue by CSEHoangV** [SAP/fundamental-ngx#13515]
### Describe your proposal

If we have some elements like `input`, `date`, or `radio` fields inside table (elements with `tabindex` defaulting to be 0), using `TAB` on keyboard does not move to the next focusable element. It looks like some elements inside table cell has `tabindex` to be `-1` and when pressing `TAB`, it jumps to the button as below:

https://github.com/user-attachments/assets/4769c56a-94b3-4cb8-a76d-588396cdf3b8

I am not quite sure on accessibility requirement of the library but I see this works fine on UI5 table:

https://github.com/user-attachments/assets/d9f010dd-b8de-440c-a915-169240cfc909

### Which versions of Angular and Fundamental Library for Angular are affected?

Angular: 15, 20, F-ngx: Latest for each angular version

### Steps to Reproduce (for bugs)

1. Create fdp-table with some elements having focusable by default.
2. Focus on one element inside table cell and use `TAB` to move to another focusable element.

### StackBlitz example(s)

Reproducible internally:

https://sap.github.io/fundamental-ngx/#/platform/table/basic#custom-columns

### Declaration

- [x] I'm not disclosing any internal or sensitive information.

### Which versions of Angular and Fundamental Library for Angular are affected?

Angular: 15, 20, F-ngx: Latest for each angular version

### Declaration

- [x] I'm not disclosing any internal or sensitive information.

**Comment by droshev** [comment]
@CSEHoangV on what version are you?

**Comment by CSEHoangV** [comment]
> [@CSEHoangV](https://github.com/CSEHoangV) on what version are you?

For Angular 15: 0.43.60
For Angular 20: 0.55.9

I updated the descriptions for versions. Issues are reproducible on both downport and main branches.

**Comment by droshev** [comment]
@mikerodonnell89 can you check this scenario is valid for the component?

**Comment by mikerodonnell89** [comment]
I do see the discrepancy between our version and ui5's, but I'm not entirely sure which way is correct. I'll ask for help from the accessibility designers

**Comment by mikerodonnell89** [comment]
The accessibility team has informed me that tab should indeed move focus to the next focusable element, like the ui5 table is doing. We can work on this
